packages/api-db-mongodb/src/db/schema.ts

```diff
@@ -42,6 +42,8 @@ export interface DBPeerProfile {
   name: string
   logoID?: string
   themeColor: string
+  callToActionURL: string
```

Comment:
I mentioned it before. You forgot to add a migration. If you already have an existing PeerProfile in the database and you run your branch you get the following error:
![image](https://user-images.githubusercontent.com/2340416/102314842-67c9cf00-3f73-11eb-92f2-69805f746de5.png)

You need to add those two fields to an existing PeerProfile in a migration. Checkout `packages/api-db-mongodb/src/migration.ts` for more information

Reply:
@roosnic1 does the `version`-number in `migration.ts` play a particular role? I'm asking since I have used for social media-metadata `version: 7` in `migration.ts` but it's not yet merged into master. So in case that `version` plays a role, @bishoymelek should maybe call his migration `version: 8` in order to prevent any conflicts with the database once our two PRs are merged into master?

Reply:
@0815pascal first come first serve.